### PR TITLE
docs: Adjust performance docs layout

### PR DESCRIPTION
**Pull request by jeromelaban**
## PR Type

What kind of change does this PR introduce?
- Documentation content changes

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/articles/Uno-UI-Performance.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Here's what to look for:
...
 	- If items content frequently change (e.g. live data in TextBlock) on iOS and Android, ListView items rendering can require the use of the `xamarin:AreDimensionsConstrained="True"` [uno-specific property](https://github.com/unoplatform/uno/blob/7355d66f77777b57c660133d5ec011caaa810e29/src/Uno.UI/UI/Xaml/FrameworkElement.cs#L86). This attribute prevents items in a list from requesting their parent to be re-measured when their properties change. It's safe to use the `AreDimensionsConstrained` property when items always have the same size regardless of bound data, and the items and list are stretched in the non-scrolling direction. If item sizes can change when the bound data changes (eg, if they contain bound text that can wrap over multiple lines, images of undetermined size, etc), or if the list is wrapped to the items, then you shouldn't set `AreDimensionsConstrained` because the list does need to remeasure itself when item data changes in that case.
 
-	You'll need to set the property on the top-level element of your item templates, as follows:
+	  You'll need to set the property on the top-level element of your item templates, as follows:
 		```xaml
```
Comment (Youssef1313) [inline doc/articles/Uno-UI-Performance.md:21]: Looks like this is the only usage of xaml as codeblock identifier, all other doc pages use xml. I suspect that xaml isn't well-recognized and won't have syntax-highlighting. Can you confirm?

Comment (davidjohnoliver) [reply doc/articles/Uno-UI-Performance.md:21]: confirmed, `xaml` isn't recognised

```suggestion
		```xml
```

**Review by davidjohnoliver — APPROVED** [review]

File: doc/articles/Uno-UI-Performance.md (hunk 1 above)
Comment (jeromelaban) [reply doc/articles/Uno-UI-Performance.md:21]: `xaml` is recognized as a specifier, how did you validate it ?

Comment (davidjohnoliver) [reply doc/articles/Uno-UI-Performance.md:21]: if you build with docfx, it will not show any highlighting

**Review by carldebilly — APPROVED** [review]

**Pull request merged**